src/app/pages/system/general-settings/support/file-ticket-form/file-ticket-form.component.ts

```diff
@@ -215,7 +215,7 @@ export class FileTicketFormComponent implements OnInit {
   openSuccessDialog(params: NewTicketResponse): void {
     const conf: DialogFormConfiguration = {
       title: this.translate.instant('Ticket'),
-      message: this.translate.instant('Congratulations! Your ticket has been submitted successfully. It may take some time before images appear.'),
+      message: this.translate.instant('Your ticket has been submitted successfully. It may take some time before images appear.'),
```

Comment:
Didn't like the happy tone? :)

Reply:
It was only a ticket. Nothing to call home about :P